salt/postgres/files/postgres-backup-to-s3.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+#
+# Backup PostgreSQL databases and upload to AWS S3
+
+set -euo pipefail
+
+PSQL_ADMIN_USER="postgres"
+
+# shellcheck disable=SC1091
+. /home/sysadmin-tools/aws-settings.local
+
+export AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION
+
+if [ "$LOGNAME" != "postgres" ]; then
```

Comment:
Isn't this script run by root's cron? (hence the `sudo -u postgres` calls)

Reply:
Fixed in new commit.